spec.md

```diff
@@ -0,0 +1,232 @@
+# Open Container Initiative
+
+## Artifacts Specification
+
+This Artifacts Spec provides a reference for artifact authors and registry implementors to support new artifact types with pre 1.0 implementations of the [distribution-spec][distribution-spec]. More particularly this repository (spec) has been tasked by the OCI TOB to serve 3 primary goals:
+
+1. **Artifact Authors** - guidance for authoring new artifact types. Including a clearing house for [well known][def-well-known-types] artifact types.
+1. **Registry Operators and Vendors** - guidance for how operators and vendors can support new artifact types, including how they can opt-in or out of well known artifact types. Registry operators that already implement `mediaType` filtering will not have to change. The artifact repo will provide context on how new `mediaTypes` can be used, and how `mediaTypes` can be associated with a type of artifact.
+1. **Clearing House for [Well-known Artifacts][def-well-known-types]** - artifact authors can submit their artifact definitions, providing registry operators a list by which they can easily support.
+
+[Artifacts v1.0][artifacts] presumes a refactoring of the [OCI Image-spec 1.0][image-spec] and the pending [OCI Distribution-spec 1.0][distribution-spec] to refactor [manifest][image-manifest] and [index][image-index] to separately versioned specs. The refactoring is for clarity of use, and is not required to support artifacts.
+
+**Reference**:
+
+- [Need a Working Group for Updating the distribution-spec #9](https://github.com/opencontainers/artifacts/issues/9)
+- [Need a Working Group for Updating the image-spec #10](https://github.com/opencontainers/artifacts/issues/10)
+
+**v1.0 of the image-spec** defines index and manifest.
+
+![image-spec](./media/image-spec-objects.png)
+
+**Artifacts 1.0** visualizes a refactoring of index and manifest.
+
+![refactored artifacts spec](./media/artifacts-spec-objects.png)
+
+## Table of Contents
+
+- [Scope](#scope)
+- [Defining OCI Artifact Types](#defining-oci-artifact-types)
+- [Defining a Unique Artifact Type](#defining-a-unique-artifact-type)
+- [Defining Supported Layer Types](#defining-supported-layer-types)
+- [Optional: Defining Config Schema](#optional-defining-config-schema)
+- [References](#references)
+
+## Scope
+
+[Artifacts v1.0][artifacts] is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifacts including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
+
+### Future Scope
+
+Future versions will support new artifact types that represent collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
```

Comment:
and here link to the below issue number(s)

Reply:
We don't yet have an issue. For the same reason you mentioned above, I'll let issues point back to the spec, rather than the spec point to an issue.